app/docs/[[...slug]]/page.tsx

```
@@ -1,61 +1,117 @@
+            ),
+            Accordion,
+            Accordions,
+            Step,
+            Steps,
+            File,
+            Folder,
+            Files,
+            blockquote: Callout as unknown as FC<ComponentProps<'blockquote'>>,
+            APIPage: openapi.APIPage,
+            DocsCategory: ({ slugs = params.slug }: { slugs?: string[] }) => (
+              <DocsCategory page={source.getPage(slugs)!} from={source} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)